src/sig_stfl/xmss/external/core_hash_n32_sha2.c

```diff
@@ -0,0 +1,11 @@
+#include <oqs/sha2.h>
+#include "hash.h"
+
+int core_hash(const xmss_params *params,
```

Comment:
How will this work when there are multiple variants using different hash functions?

Reply:
I think I will use define macro as inside the file and cmakelist https://github.com/open-quantum-safe/liboqs/pull/1482/commits/6712094df456542d5d1386189b3eb49c4b9f59f8 and https://github.com/open-quantum-safe/liboqs/pull/1482/commits/3677849a77ab5b098a672e17734e939e85cc54ae